sdk/ai/azure-ai-client/azure/ai/client/operations/_patch.py

```diff
@@ -1127,6 +1127,231 @@ def upload_file(
 
         raise ValueError("Invalid parameters for upload_file. Please provide the necessary arguments.")
 
+    @overload
+    def upload_file_and_poll(self, body: JSON, sleep_interval: float = 1, **kwargs: Any) -> _models.OpenAIFile:
+        """Uploads a file for use by other operations.
+
+        :param body: Required.
+        :type body: JSON
+        :keyword sleep_interval: Time to wait before polling for the status of the uploaded file. Default value
+         is 1.
+        :paramtype sleep_interval: float
+        :return: OpenAIFile. The OpenAIFile is compatible with MutableMapping
+        :rtype: ~azure.ai.assistants.models.OpenAIFile
```

Comment:
please change rtype
:rtype: ~azure.ai.assistants.models.OpenAIFile

to

:rtype: ~azure.ai.client.models.OpenAIFile

In all places where you have azure.ai.assistants.models.OpenAIFile

Reply:
Done